src/dune_rules/coq_stanza.ml

```diff
@@ -60,6 +61,8 @@ module Buildable = struct
       located
         (field "mode" ~default
            (Dune_lang.Syntax.since coq_syntax (0, 3) >>> Coq_mode.decode))
+    and+ use_stdlib =
+      field ~default:true "stdlib" (Dune_lang.Decoder.(enum [ ("yes", true); ("no", false) ]))
```

Comment:
```suggestion
    and+ use_stdlib =
      field ~default:true "stdlib" (Dune_lang.Syntax.since coq_syntax (0, 6) >>> Dune_lang.Decoder.(enum [ ("yes", true); ("no", false) ]))
```
Shouldn't this be part of a new syntax version?

Reply:
Yes, once we document it; originally this PR didn't include this change, but I added to test tactician.

If you are OK with it, we can merge this PR as is, and we can test and document the feature in #6164

Reply:
Sounds good.